Dataset/schema.json

```diff
@@ -43,13 +43,41 @@
             "description": "Property. Every contact element"
           }
         },
+        "belongsToCatalogue": {
+          "description": "Relationship. It links the Dataset to its parent Catalogue. Model:'https://www.w3.org/ns/dcat#Catalogue'",
+          "anyOf": [
+            {
+              "type": "string",
+              "minLength": 1,
+              "maxLength": 256,
+              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
+              "description": "Property. Link to the catalogue"
+            },
```

Comment:
is this compatible with the attribute being a relationship?

Reply:
This is something I usually see in relationships defined within Smart Data Models. For instance https://github.com/smart-data-models/dataModel.Environment/blob/a734b49a8c557d1355203500c476e826005e9f6a/AirQualityObserved/schema.json#L150-L166 So I have always defined relationships like that. Is it wrong?

Reply:
Well, don't really know, I was just a bit surprised :) I guess @albertoabellagarcia will have the answer.

Reply:
Regarding compatibility yes, this is 100% compatible because it is allowing the user of SDM with NGSIv2. But whenever there is that option with this pattern there is another option with a simple uri which is valid for NGSILD

Reply:
In order to be accepted it needs to document that this attribute does not belong to the current version of DCAT-AP 2.1.1